.github/workflows/test_python_versions.yml

```diff
@@ -0,0 +1,61 @@
+name: Run Integration Tests Python Versions
```

Comment:
Instead of having a separate workflow, why don't we add postgres to daily test_profiles?

Reply:
I think that is a good idea. I'll do it

Reply:
We did lose testing different dbt versions against different python versions. But on second thought, I don't think this adds much value, since the dbt code is already tested by dbt-labs/dbt-core repo.